pkg/template/template.go

```diff
@@ -10,12 +10,20 @@ import (
 	"github.com/cyberark/conjur-oss-suite-release/pkg/changelog"
 )
 
-// UnifiedChangelogTemplateData stores all the data needed for generation of
-// templates
-type UnifiedChangelogTemplateData struct {
+// Component represents a suite component with all of its changelogs and relevant
+// pin data
+type Component struct {
+	Repo        string
+	Changelogs  []*changelog.VersionChangelog
+	ReleaseName string
+	ReleaseDate string
+}
+
+// SuiteData stores all the data needed for generation of templates in the suite
+type SuiteData struct {
```

Comment:
nit: I wonder if it would be useful to name this as the thing it represents/is, `ReleaseSuite`. I thought that the Data suffix might be redundant because this is a struct definition.